File: basic/src/main/java/com/yuanshenbin/basic/manager/BusObserver.java
package com.yuanshenbin.basic.manager;


import io.reactivex.Observer;

/**
 * author : yuanshenbin
 * time   : 2018/10/25
 * desc   :
 */

public abstract class BusObserver<T> implements Observer<T> {


    @Override
    public void onError(Throwable e) {

    }

    @Override
    public void onComplete() {

    }
}

File: basic-kotlin/src/main/java/com/yuanshenbin/basic/manager/BusObserver.kt
package com.yuanshenbin.basic.manager

import io.reactivex.Observer

/**
 * author : yuanshenbin
 * time   : 2018/10/25
 * desc   :
 */
abstract class BusObserver<T> : Observer<T> {
    override fun onError(e: Throwable) {}
    override fun onComplete() {}
}